descqa/NumberDensityVersusRedshift.py

```diff
@@ -58,9 +58,9 @@ class NumberDensityVersusRedshift(BaseValidationTest):
     validation_color = 'black'
     default_markers = ['o', 'v', 's', 'd', 'H', '^', 'D', 'h', '<', '>', '.']
 
-    def __init__(self, z='redshift_true', band='i', N_zbins=44, zlo=0., zhi=1.1,
+    def __init__(self, z='redshift_true', band='i', N_zbins=10, zlo=0., zhi=1.1,
                  observation='', mag_lo=27, mag_hi=18, ncolumns=2, normed=True,
-                 jackknife=False, N_jack=10, ra='ra', dec='dec', pass_limit=2., use_diagonal_only=True,
+                 jackknife=False, N_jack=20, ra='ra', dec='dec', pass_limit=2., use_diagonal_only=True,
```

Comment:
Good to change the defaults to be more reasonable.  I would suggest adding a comment indicating to users what are the considerations for the choice of `N_zbins` and `N_jack`.

Reply:
added docstring